umap/static/umap/js/umap.js

```diff
@@ -352,7 +353,7 @@ L.U.Map.include({
       document.body,
       'umap-caption-bar-enabled',
       this.options.captionBar ||
-        (this.options.slideshow && this.options.slideshow.active)
+      (this.options.slideshow && this.options.slideshow.active)
```

Comment:
what is the `slideshow` object and how can we make sure it has the `active` property?